File: Beginner_Projects/Patterns/Alphabats/C++/J.cpp
#include <iostream>
#include <conio.h>
using namespace std;

int main()
{
    for (int row = 0; row < 7; row++)
    {
        for (int col = 0; col < 9; col++)
        {
            if (row == 0 || (col == 7 && row < 5) || (col == 6 && row == 5) || ((col == 4 || col == 3 || col == 2) && row == 6) || ((row == 5 || row == 4) && col == 0))
            {
                cout << "*";
            }
            else
            {
                cout << " ";
            }
        }
        cout << endl;
    }
    getch();
}

// Created by Salfi Hacker Mansoor Bukhari

File: Beginner_Projects/Patterns/Alphabats/Python/J.py
for row in range(7):
    for col in range(7):
        if(row == 0 or (col == 5 and (row < 5 and row != 0)) or (col == 4 and row == 5) or ((col >= 1 and col < 4) and row == 6) or ((row == 5 or row == 4) and col == 0)):
            print("*", end=" ")
        else:
            print(end="  ")
    print()
